### PR TITLE
Add LilyGO TTGO LORA32 image.

### PR DESCRIPTION
**Pull request by APIUM**
Signed-off-by: Algy Tynan <algy@tynan.io>

### CODE REVIEW

**Comment by APIUM** [comment]
Images are from the manufacturer. Board definition has already been merged.

**Comment by APIUM** [comment]
Also, clicking on the board on the [micropython download page](https://micropython.org/download/) directs the user to the generic ESP32 firmware - this doesn't happen for other boards that are missing an image, what might be the cause of this?

**Comment by mattytrentini** [comment]
> Is there a suggested image size for images in this repo? 

Only by file size as far as I'm aware; they should be no larger than 200KB (at ~158KB this one is _ideal_). The image will be scaled down by the Download web application for the thumbnail on the front page.

> Also, clicking on the board on the micropython download page directs the user to the generic ESP32 firmware - this doesn't happen for other boards that are missing an image, what might be the cause of this?

There's a minor error in the [LILYGO_TTGO_LORA32/board.json](https://github.com/micropython/micropython/blob/master/ports/esp32/boards/LILYGO_TTGO_LORA32/board.json#L14) - just need to remove the `id` field. It's an optional field to specify the URL stub used by the Download web app. If it's _not_ specified the board name will be used.

In other words, currently the download URL for `LILYGO_TTGO_LORA32` is:
```
https://micropython.org/download/esp32/
```
Removing `id` will change it to:
```
https://micropython.org/download/LILYGO_TTGO_LORA32/
```

In any case, this image looks good and the `board.json` is correctly configured to pick it up - @dpgeorge I think this can be merged! 

We'll also need a tiny PR to remove the `id` field on the main repo.

**Comment by APIUM** [comment]
@mattytrentini Thanks, that fix is up here: https://github.com/micropython/micropython/pull/8650
